openssl/src/dh.rs

```diff
@@ -57,7 +97,11 @@ impl Dh {
         }
     }
 
+    /// return a DH object for the IETF RFC 5114 value.
```

Comment:
See above

Reply:
Am I to assume this method is for the 2047 bit MODP group with 224-bit prime order subgroup (and so on)? I want to be sure since this is well outside of my realm of knowledge right now.

Reply:
Yeah, they all come from that same RFC, just with different sizes.